{{cookiecutter.project_name}}/.github/workflows/tests.yml

```diff
@@ -97,14 +97,17 @@ jobs:
 
       - name: Upload coverage data
         if: always() && matrix.session == 'tests'
-        uses: "actions/upload-artifact@v3"
+        uses: actions/upload-artifact@v4
         with:
-          name: coverage-data
+{% raw %}
+          name: coverage-data-${{ matrix.python }}-${{ matrix.os }}
+{% endraw %}
```

Comment:
Why are these % raw tags needed?

Reply:
Otherwise the line would look like this:
```
          name: coverage-data-${{"{{"}} matrix.python {{"}}"}}-${{"{{"}} matrix.os {{"}}"}}
```
This was suggested by @webknjaz

Reply:
@bosd it's usually cleaner to wrap chunks of mustache that shouldn't be rendered this way. I find that seeing `{{ '{{' }}` all over the place doesn't help readability.